### PR TITLE
Added different Payment plan options

### LINKED ISSUE #186: Develop Pro Sections with Payment Methods and Subscription Plans
aim to enhance the Codeteria platform by implementing a dedicated "Pro" section that offers users premium features through subscription plans. This enhancement will include:

Pro Section Pages: Design and develop engaging and informative pages that outline the benefits of the Pro membership, including exclusive features and user testimonials.

Payment Integration: Implement secure payment methods (e.g., Stripe, PayPal) to facilitate smooth transactions for users opting for Pro plans.

Subscription Plans: Create various subscription tiers (monthly, yearly) with clear pricing and feature comparisons to help users choose the best plan for their needs.

### PR DESCRIPTION
**Pull request by shivenyadavs**
closes #186 
Payment Plans Section: Below the "Coming Soon" part, I added a section that includes three payment options: Monthly Pro, Yearly Pro, and College Plan.
Buttons: Each plan has a corresponding button (Subscribe now or Buy now), which navigates to specific routes (/payment/monthly-pro, /payment/yearly-pro, /payment/college-plan). You'll need to handle these routes in your app's routing logic.
Styling: I used similar Tailwind classes to maintain the design consistency.

### CODE REVIEW

**Comment by shivenyadavs** [comment]
@arjunatapadkar can you please check the changes which i made.

**Comment by shivenyadavs** [comment]
Screenshot (71)
here is the code snap shot

**Comment by arjunatapadkar** [comment]
Snapshots of ui will be more helpful

**Review by arjunatapadkar — CHANGES_REQUESTED** [review]
Don't change upcoming page 

Create a new Page for Pro

**Comment by shivenyadavs** [comment]
@arjunatapadkar  i made the changes can you please check it

**Comment by arjunatapadkar** [comment]
And share snapshot of ui for pro page

File: src/pages/Homepage.jsx
Comment (arjunatapadkar) [inline src/pages/Homepage.jsx:1]: page: "/pro"

**Comment by shivenyadavs** [comment]
@arjunatapadkar  added page pro and made changes in the Homepage.jsx

**Review by arjunatapadkar — APPROVED** [review]

**Pull request merged**